packages/idle-timer/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@solid-primitives/idle-timer",
```

Comment:
Originally I thought that having a pkg called just `idle` for primitives that tracked the user's activity would be confusing in case any primitive wrapping the `window.requestIdleCallback()`, which has a different use, were to be created. So I thought there should be a pkg for browser-idle related primitives, and one for user-idle related ones (like this one, or a wrapper of the API you mentioned). To my non-native speaker ear, `user-idle` didn't sound natural and I opted for `idle-timer`. Later I realized that the `createDeferred` primitive does what `window.requestIdleCallback()` is set to do (although I am not 100% sure I got it right), and so there would be no need for a `browser-idle` pkg and the name `idle` could be reserved for the pkg encompassing user idle detection primitives; but I forgot to change the name, which I'll do now.
Would just changing this line in suffice or should I also change it somewhere else?

Reply:
ok I changed the pkg name to `idle` in all of the occurrences that I could find, please let me know if I missed anything

Reply:
all good
search and replace is how we do it :)